covid19-dashboard/Covid19.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from send_request import send_request
+from collections import defaultdict
+
+
+class Covid19:
+    def __init__(self, api_key) -> None:
+        self._geo_exceptions = ['geoId/3651000', 'geoId/2938000']
```

Comment:
what are the exceptions? If there is data issue then should fix it. For now, it's good to add a comment.

Reply:
I have removed the exceptions as they are included in the list of dcids returned.